### DISCUSSION TITLE
What is your opinion?

### DISCUSSION THREAD

**Discussion by vitalspace** [enable3d/enable3d#99] [Help / Questions]
What is your opinion on this repository. more specifically this example.  [ASWebGLue](https://github.com/battlelinegames/ASWebGLue/tree/master/src/examples/Obj)  Basically it allows us to load ".obj" files through Assemblyscript.

![Image of monkey.obj](https://i.imgur.com/17AhcBD.png)

**Comment by vitalspace** [comment]
🤔

**Comment by yandeu** [comment]
No idea. Have never used `.obj` files before :/

**Comment by vitalspace** [comment]
mmm okay. at some point do you think enable 3d has integration with "Assemblyscript"?
